IdentityCore/src/requests/MSIDTokenResponseHandler.m

```diff
@@ -89,6 +92,16 @@ - (void)handleTokenResponse:(MSIDTokenResponse *)tokenResponse
         return;
     }
     
+#if !AD_BROKER
```

Comment:
nit: This does not sound like part of the token handler, is it possible to move the logic into `MSIDDeviceInfo`

Reply:
Updated